man/repart.d.xml

```diff
@@ -439,7 +434,9 @@
         <citerefentry project='man-pages'><refentrytitle>mkfs.xfs</refentrytitle><manvolnum>8</manvolnum></citerefentry>
         due to limitations of its protofile format.</para>
 
-        <para>This option cannot be combined with <varname>CopyBlocks=</varname>.</para>
+        <para>When this option is used in combination with <varname>CopyBlocks=</varname>,
+        <command>systemd-repart</command> will first try the <varname>CopyBlocks=</varname> logic. If that is
+        not possible, it falls back to the <varname>CopyFiles=</varname> logic.</para>
```

Comment:
Maybe better to mention that `CopyFiles=` will be ignored when `CopyBlocks=` works.